templates/new_device_login.txt

```diff
@@ -5,8 +5,8 @@
 {{#if location}}{{ location }}{{/if}}
 {{#if timestamp}}{{ timestamp }}{{/if}}
 
-{{t "This is an automated email; if you didn't add a new device to your Firefox Account, you should change your password immediately at %(resetLink)s."}}
-{{t "For more information, please visit %(supportUrl)s"}}
+{{{t "This is an automated email; if you didn't add a new device to your Firefox Account, you should change your password immediately at %(resetLink)s."}}}
```

Comment:
Wow, I'm suprised we didn't notice this before. The link would have been HTML escaped.